### ISSUE TITLE
Add a background music for the onload screen.

### ISSUE THREAD

**Issue by prantikseal** [subhamgcon/Fight-game#6]
On loading the website there will be a background music can have a mute functionality as well

The music will be non-copywrited

**Comment by subhamgcon** [comment]
@prantikseal That's a great idea, Can you also add a loading animation which will be retro-styled?

**Comment by prantikseal** [comment]
Will have to figure out. I can try. can you assign it to me? @subhamgcon

**Comment by prantikseal** [comment]
if possible please give a tag of hacktoberfest to this  issue :D if I get assigned :)

**Comment by subhamgcon** [comment]
@prantikseal Done. All the best!

**Comment by prantikseal** [comment]
Thanks!

**Comment by subhamgcon** [comment]
@prantikseal Are you working on it?

**Comment by prantikseal** [comment]
Yes, I am please give me a day or two will give and update very soon @subhamgcon

**Comment by subhamgcon** [comment]
Hi @prantikseal, Any updates?
